pupil_src/shared_modules/pi_preview/sensor.py

```diff
@@ -0,0 +1,118 @@
+import logging
+from pyglui import ui
+
+from pi_preview import GAZE_SENSOR_TYPE, Linked_Device
+
+logger = logging.getLogger(__name__)
+
+
+class GazeSensor:
+    get_data_timeout = 100  # ms
+
+    def __init__(self, network, linked_device):
+        self._ndsi_sensor = None
+        self.network = network
+        self.host_uuid = linked_device.uuid
+        self.host_name = linked_device.name
+        self.offset = [0, 0]
+        if self.linked:
+            self.activate()
+
+    @property
+    def linked(self):
```

Comment:
nitpick: I find prefixing `bool` properties with `is_XYZ` much more readable, especially since in some contexts it even makes sense to have both `XYZ()` and `is_XYZ` (e.g. `my_array.sorted()` returns a new, sorted copy of `my_array`, while `my_array.is_sorted` returns `True` if `my_array` is already sorted).